api/db-cdr/generate-cdr/build-prep-survey.sh

```diff
@@ -239,4 +240,4 @@ echo "Loading data into prep_survey"
 bq load --project_id="$BQ_PROJECT" --source_format=CSV "$BQ_DATASET.prep_survey" \
 "gs://$BUCKET/$BQ_DATASET/cdr_csv_files/$OUTPUT_FILE_NAME" "$SCHEMA_PATH/prep_survey.json"
 
-rm -rf $TEMP_FILE_DIR
\ No newline at end of file
+rm -rf $TEMP_FILE_DIR
```

Comment:
Why did we change this line?

Reply:
hmm I did not change that
![image](https://user-images.githubusercontent.com/4452480/160864774-766b2c83-1161-484a-b63d-62e19e069e1e.png)